.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+<!--
+Thanks for sending a pull request!  Here are some tips for you:
+  1. If this is your first time, please read our contributor guidelines: https://github.com/delta-incubator/delta-kernel-rs/blob/main/CONTRIBUTING.md
+  2. Ensure you have added or run the appropriate tests for your PR and run `cargo fmt`.
+  3. If the PR is unfinished, add '[WIP]' in your PR title, e.g., '[WIP] Your PR title ...'.
+  4. Be sure to keep the PR description updated to reflect all changes.
+-->
+
+### What changes are proposed in this pull request?
+<!--
+Please clarify what changes you are proposing and why the changes are needed.
+The purpose of this section is to outline the changes, why they are needed, and how this PR fixes the issue.
+If the reason for the change is already explained clearly in an issue, then it does not need to be restated here.
+  1. If you propose a new API or feature, clarify the use case for a new API or feature.
+  2. If you fix a bug, you can clarify why it is a bug.
+  3. If possible, provide a concise example to reproduce the issue for a faster review.
```

Comment:
should this be here or in the issue template?

Reply:
Yeah this was part of spark one, but you're right that it makes more sense in an issue